src/mixins/filing-template-mixin.ts

```diff
@@ -567,10 +603,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       })
     }
 
-    // Store people and roles
-    if (filing.incorporationApplication) { // *** FUTURE: expand for other filing types
-      this.setPeopleAndRoles(filing.incorporationApplication.parties || [])
-    }
+    // Store people and roles *** FUTURE: expand for other filing types
+    if (filing.incorporationApplication) this.setPeopleAndRoles(filing.incorporationApplication.parties || [])
```

Comment:
There's too much going on in this line. Please break it up using {}.

Reply:
Will do